setup.py

```diff
@@ -86,15 +86,34 @@ def is_platform_mac():
     if not _CYTHON_INSTALLED:
         raise ImportError('No supported version of Cython installed.')
     from Cython.Distutils import build_ext as _build_ext
+    from Cython import Tempita as tempita
```

Comment:
agree with @shoyer here, though as I said above no prob with bumping cython dep required version if needed; we have 0.19.1 which is pretty old.